app/common/topBar.controller.js

```diff
@@ -0,0 +1,18 @@
+{
+	function topBarController (UtilsService) {
```

Comment:
poza tym, zamiast robic function i w srodku robic this. etc. sprobuj skorzystac z class